core/src/main/java/jenkins/util/SystemProperties.java

```diff
@@ -146,25 +146,43 @@ public static String getString(String key) {
      * @exception  IllegalArgumentException if {@code key} is empty.
      */
     public static String getString(String key, @CheckForNull String def) {
+        return getString(key, def, Level.CONFIG);
+    }
+
+    /**
+     * Gets the system property indicated by the specified key, or a default value.
+     * This behaves just like {@link System#getProperty(java.lang.String, java.lang.String)}, except
+     * that it also consults the {@link ServletContext}'s "init" parameters.
+     *
+     * @param      key   the name of the system property.
+     * @param      def   a default value.
+     * @param      logLevelIfAbsent the level of the log if the provided key is not found.
+     * @return     the string value of the system property,
+     *             or {@code null} if the the property is missing and the default value is {@code null}.
+     *
+     * @exception  NullPointerException if {@code key} is {@code null}.
+     * @exception  IllegalArgumentException if {@code key} is empty.
+     */
+    public static String getString(String key, @CheckForNull String def, Level logLevelIfAbsent) {
         String value = System.getProperty(key); // keep passing on any exceptions
         if (value != null) {
-            if (LOGGER.isLoggable(Level.CONFIG)) {
-                LOGGER.log(Level.CONFIG, "Property (system): {0} => {1}", new Object[] {key, value});
+            if (LOGGER.isLoggable(logLevelIfAbsent)) {
```

Comment:
Right. Fixed. Switch back to simply logLevel for every places, since anyway ...IfAbsent is not the whole picture: the log will also be used for impossible conversions, not just when it's not specified and so on.